shared/constants/chat2/message.js

```diff
@@ -1080,7 +1082,7 @@ export const shouldShowPopup = (state: TypedState, message: Types.Message) => {
     case 'sendPayment': {
       // Is the payment pending?
       const paymentInfo = getPaymentMessageInfo(state, message)
-      if (!paymentInfo || ['cancelable', 'pending', 'canceled'].includes(paymentInfo.get('status'))) {
+      if (!paymentInfo || ['claimable', 'pending', 'canceled'].includes(paymentInfo.get('status'))) {
```

Comment:
This `if` statement doesn't seem to match the "Is the payment pending?" comment.